src/components/Map/components/MarkersDrawer/components/Markers/components/Marker/Marker.js

```diff
@@ -51,6 +51,22 @@ const Marker = ({ markerData = {}, markerOptions = {}, readOnly = true }) => {
 		};
 	};
 
+	const animate = () => {
```

Comment:
Esto lo podemos mejorar para evitar if anidados.

```suggestion
	const animate = () => {
		if (!markerRef.current) return;

		markerRef.current?.marker?.setAnimation(window.google.maps.Animation[animation?.name]);

		if (animation?.duration && isNumber(animation?.duration)) {
			setTimeout(() => {
				markerRef.current?.marker?.setAnimation(null);
			}, animation?.duration);
		}
	};
```

Reply:
@colomfernando  PR actualizado!